### DISCUSSION TITLE
How to best contribute changes

### DISCUSSION THREAD

**Discussion by laat0003** [modflowpy/flopy#1288] [Q&A] [answered]
Hi,

Is there a "how to" or "recommended approach" that someone can follow to recommend or make changes to the code? Are pull requests the only way to contribute changes? Say I stumble across a problem that is an easy-fix one liner - what is the best or preferred thing to do in this instance and what, if any,  are the alternative methods.

Cheers for you time.

**Comment by jlarsen-usgs** [comment, marked as answer]
@laat0003 

We only accept pull requests to FloPy. The reason for this is to provide a space for 1) automated testing to make sure changes didn't introduce unintended effects, 2) reviewing the proposed changes, and 3) ensuring that the changes follow the code of conduct.

The [Contributing page](https://github.com/modflowpy/flopy/blob/develop/CONTRIBUTING.md) lays out the guidelines for making changes to the code.
